src/Microsoft.AspNet.Mvc.ApiExplorer/DefaultApiDescriptionProvider.cs

```diff
@@ -302,6 +313,56 @@ private string GetRelativePath(RouteTemplate parsedTemplate)
             return string.Join("/", segments);
         }
 
+        private IReadOnlyList<ApiRequestFormat> GetRequestFormats(
+            ControllerActionDescriptor action,
+            IApiRequestMetadataProvider[] requestMetadataAttributes,
+            Type type)
+        {
+            var results = new List<ApiRequestFormat>();
```

Comment:
Fair enough, my comment was more due to the fact that in other parts of the code base we return null instead of creating a list (specially on public extensibility points) but I'm happy with returning an empty list to. I'm all for avoiding null checks where possible, but we do it in that way on other parts of the product, hence the comment.